eslint.config.js

```diff
@@ -93,7 +94,7 @@ export default [
 	},
 	{
 		files: ["src/rules/*.js"],
-		...eslintPluginRulesRecommendedConfig,
+		extends: [eslintPluginRulesRecommendedConfig],
 		rules: {
 			...eslintPluginRulesRecommendedConfig.rules,
```

Comment:
```suggestion
```

No need for this anymore since the rules key from this config object doesn't overwrite the rules key from the plugin's config object.